pkg/network/node/node.go

```diff
@@ -135,9 +133,7 @@ func New(c *OsdnNodeConfig) (*OsdnNode, error) {
 		policy = NewMultiTenantPlugin()
 		pluginId = 1
 		// Userspace proxy is incompatible with conntrack.
-		if c.ProxyMode != kubeproxyconfig.ProxyModeUserspace {
-			useConnTrack = true
-		}
+		useConnTrack = true
```

Comment:
actually, if this affects any users, then it requires bumping `ruleVersion` too (since previous multitenant+userspace users would now need conntrack-aware flows rather than non-conntrack flows)

Reply:
good point bumped up the ruleVersion